jax/random.py

```diff
@@ -116,8 +116,9 @@
    identical across JAX/XLA versions    ✅        ✅
    =================================   ========  =========  ===  ==========  =====  ============
 
-(*): with jax_threefry_partitionable=1 set
-(**): with XLA_FLAGS=--xla_tpu_spmd_rng_bit_generator_unsafe=1 set
+(*): with `jax_threefry_partitionable=1` set
+
+(**): with `XLA_FLAGS=--xla_tpu_spmd_rng_bit_generator_unsafe=1` set
```

Comment:
For restructured text, single backticks are rendered as itallic, while double backticks are rendered as code. I think we want double backticks here.

Reply:
Corrected, thanks!